tests/test_e2e.py

```
@@ -1254,3 +1256,240 @@ def test_list_re_replace():
+    assert res.result_set == [[1]]
+
+    res = query("RETURN abs(0) AS name")
+    assert res.result_set == [[0]]
+
+    res = query("RETURN abs(null) AS name")
+    assert res.result_set == [[None]]
+
+    for value, name in [(True, 'Boolean'), (False, 'Boolean'), ({}, 'Map'), ([], 'List')]:
+        try:
+            query(f"RETURN abs({value}) AS r")
+            assert False, "Expected an error"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)